.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+
+language: go
+go:
+- 1.5
```

Comment:
People might still build with `go 1.5`, so I would build with `1.5.4` and `1.6.2`.
I don't really have a opinion about `tip`, is a `devel` version really useful ?